sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DeadStores.cs

```diff
@@ -758,5 +758,20 @@ public int Start()
         }
 
         public void Archive() {}
+
+        public class TestClass

```

Comment:
This new test class is nested inside a test class for a different bug (`public class FooBarBaz`), which is confusing.
It should be top-level class, and it would be useful to have a link to the bug (same as in line 737)